src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,265 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Scanner;
+
+import static duke.FileManager.*;
```

Comment:
Are there more specific classes from `FileManager `that you would be importing. It would be good to state them explicitly here.